.devcontainer/Dockerfile

```diff
@@ -62,13 +64,12 @@ RUN sudo bash -c 'curl https://get.wasmer.io -sSfL | sudo WASMER_DIR=/home/vscod
 RUN sudo bash -c 'mkdir -p /home/vscode/.local/juvix'
 RUN sudo bash -c 'chmod -R a+rXw /home/vscode/.local/juvix'
 
-RUN sudo bash -c 'curl -s https://api.github.com/repos/anoma/juvix/releases/latest \
-      | grep -a "browser_download_url.*linux" \
-      | cut -d : -f 2,3 \
-      | tr -d \" \
-      | wget --output-document juvix.zip -qi - \
-      && unzip juvix.zip \
-      && mv juvix /home/vscode/.local/bin/juvix'
+
+RUN sudo bash -c 'curl --proto "=https" --tlsv1.2 -sSf https://raw.githubusercontent.com/anoma/juvix-installer/main/juvix-installer.sh \
```

Comment:
```suggestion
RUN sudo bash -c 'curl --proto "=https" --tlsv1.2 -sSf https://get.juvix.org \
```

Reply:
With this suggestion, you need to include the `-L` option to curl because https://get.juvix.org is a redirect.